samples/snippets/list_clusters.py

```diff
@@ -49,8 +49,10 @@ def main(project_id, region):
     else:
         # Use a regional gRPC endpoint. See:
         # https://cloud.google.com/dataproc/docs/concepts/regional-endpoints
-        client_transport = cluster_controller_grpc_transport.ClusterControllerGrpcTransport(
-            address="{}-dataproc.googleapis.com:443".format(region)
+        client_transport = (
+            cluster_controller_grpc_transport.ClusterControllerGrpcTransport(
+                address="{}-dataproc.googleapis.com:443".format(region)
+            )
         )
         dataproc_cluster_client = dataproc_v1.ClusterControllerClient(client_transport)
```

Comment:
No need to block this PR on this, but there's an easier way to set regional endpoints now through `google.api_core.client_options`. See this speech sample: https://cloud.google.com/speech-to-text/docs/endpoints